### PR TITLE
docs: update favicon.ico

### PR DESCRIPTION
**Pull request by avasisht23**
image

---

## PR-Codex overview
This PR focuses on updating a link in the `demos.md` file. 

### Detailed summary
- Updated the link to the gasless NFT minter demo in `demos.md`.

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

**Comment by avasisht23** [comment]
Current dependencies on/for this PR:
* main
  * **PR #169** <a href="https://app.graphite.dev/github/pr/alchemyplatform/aa-sdk/169" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/alchemyplatform/aa-sdk/169?utm_source=stack-comment).

File: site/demos.md [hunk 1]
```diff
@@ -25,4 +25,4 @@ This demo shows how to mint an ERC-20 token to a Light Account using Alchemy's G
 
 This demo shows how to mint an NFT (ERC-721 token) to a Light Account using Alchemy's Gas Manager to provide a gas-less minting experience.
 
-[Try it out!](https://gasless-nft-minter-v2.vercel.app/)
\ No newline at end of file
+[Try it out!](https://gasless-nft-minter-v2.vercel.app/)
```
Comment (moldy530) [inline site/demos.md:28]: what's changed here 👀

**Review by moldy530 — APPROVED** [review]
LGTM

File: site/demos.md (hunk 1 above)
Comment (avasisht23) [reply site/demos.md:28]: linting, needed a newline

**Pull request merged**